src/RequestHandler/outofprocess/forwardinghandler.cpp

```diff
@@ -314,15 +310,7 @@ FORWARDING_HANDLER::OnExecuteRequestHandler()
     m_RequestStatus = FORWARDER_DONE;
 
     //disbale client disconnect callback
-    if (m_pDisconnect != NULL)
-    {
-        if (fHandleSet)
-        {
-            m_pDisconnect->ResetHandler();
-        }
-        m_pDisconnect->CleanupStoredContext();
```

Comment:
You are losing this CleanupStoredContext when switching to RemoveRequest. Is that acceptable?

Reply:
This looks like a bug in the original code (before the change) to me. My understanding is that the class that implements IHttpStoredContext and its CleanupStoredContext method should not call the method. Instead, CleanupStoredContext should only be called once by iiscore at the end of the lifetime of the context container, otherwise a double cleanup may cause an AV.

Reply:
yes. no need to do cleanstoredcontext